docs/tutorials/delete-a-user-levibeverly.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: page
+---
+
+# Tutorial: Delete a user
+
+Use the following steps to quickly identify and delete a user from your subscriber base. You can perform these steps using [cURL commands](#delete-a-user-with-curl-commands) or in the [Postman application](#delete-a-user-in-postman).
+
+*Note: This tutorial assumes that you have already completed the To-Do Service first-time setup process. If you have not, complete the steps described in [First-time setup](), then return to this tutorial.*
```

Comment:
```suggestion
_**Note:** This tutorial assumes that you have already completed the To-Do Service first-time setup process. If you have not, complete the steps described in [First-time setup](../before-you-start-a-tutorial.md), then return to this tutorial._
```
A pull request should result in a finished document that's ready for customers to use. Go ahead and add the links.

My linter (and I) prefer using underscore for emphasis (italics) to distinguish it from bold, which uses the double asterisk `**`. Bold-italics become `_**` which seems easier to spot in the code than `***` which could be bold-italics or a bold list item with a missing space.